dev/TabView/TabViewItemAutomationPeer.cpp

```diff
@@ -43,3 +51,37 @@ winrt::hstring TabViewItemAutomationPeer::GetNameCore()
 
     return returnHString;
 }
+
+
+bool TabViewItemAutomationPeer::IsSelected()
+{
+    if (auto tvi = Owner().try_as<TabViewItem>())
+    {
+        const bool returnValue = tvi->IsSelected();
+        return returnValue;
+    }
+    return false;
+}
+
+winrt::IRawElementProviderSimple TabViewItemAutomationPeer::SelectionContainer()
+{
+    return nullptr;
```

Comment:
You will need to return the TabView's automation peer here.

Reply:
So we need a reference to TabView here anyway?

Reply:
Good point. We need to either pass it down to the item or look up the tree from the item in this case. That said, I'm not exactly sure how Narrator uses this method though.

Reply:
I've added the logic back, I added earlier in this PR. Walking up the tree seems a bit expensive here, though I am not a fan of having this relation between TabViewItem and TabView just for this.

Reply:
@YuliKl is there someone we can ask about how Narrator uses this?

Reply:
Narrator uses `SelectionContainer` for a few purposes:
1. Listening to UIA events for the entire container.
2. Understanding how selection behaves in the container and therefore which actions should be presented to the user. For instance, if the container is `single select + required`, Narrator will not expose the action of removing an item from selection.
3. Making announcements that inform the user about selection state of individual items. For instance, Narrator's "selected" for lists depends on whether the container supports multi-selection. Not quite your case but it does not mean it cannot become one.

Does that help?

Reply:
I think point 2 is a good reason here to not provide a `nullptr` for the `SelectionContainer`. After all TabView does not allow deselecting an item, and without the `SelectionContainer` being provided, Narrator wouldn't be able to hide that option from users.

Thank you for the information @LukaszMakar !